akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/HttpEncoding.scala

```diff
@@ -55,7 +55,6 @@ object HttpEncodings extends ObjectRegistry[String, HttpEncoding] {
   val deflate          = register("deflate")
   val gzip             = register("gzip")
   val identity         = register("identity")
-  val `identity;q=MIN` = identity.withQValue(Float.MinPositiveValue)
```

Comment:
OK, so `identity;q=MIN` is gone completely. The most general thing being identity now, right?